gradle.properties

```diff
@@ -1,4 +1,4 @@
 version=23.1.0-SNAPSHOT
 
 org.gradle.welcome=never
-org.gradle.jvmargs=-Xmx1g
+org.gradle.jvmargs=-Xmx2g
```

Comment:
is this intended to be in this PR?

Reply:
removed, is handled in https://github.com/hyperledger/besu/pull/4932/files